src/styles/_components/_header.scss

```diff
@@ -313,13 +347,20 @@ $header-height: 60px;
         }
 
         &--active {
-            opacity: 1;
             transform: translate3d(0, 0%, 0);
+            transition-delay: 0s;
+
+            &:before {
+                transform: translate3d(0, 0px, 0);
```

Comment:
Above you use ```%``` instead of ```px```. Is there any reason for this? I believe we should keep the same: either px or %